packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -27,7 +27,7 @@ export interface StartFlags {
 /**
  * DeFiChain defid node managed in docker
  */
-export abstract class DeFiDContainer extends DockerContainer {
+export class DeFiDContainer extends DockerContainer {
```

Comment:
My preference will be keep this abstract, and it seems feasible to me, by change all the direct `DeFiDContainer` usage to `RegTestContainer`. Generally speaking everything under testing package should use RegTestContainer instead of DeFiDContainer

Reply:
The default 8554 port made it more error prone.

Reply:
change to `RegTestContainer` will bypass `testnet`, `mainnet` containers. But not sure if we are going to write tests for these networks anytime soon or never. have to make a decision.

Reply:
Yup, IMO not trying to cater some future possibility, so vote for not bothering other network for now. The logic written is loading the hardcoded regtest elliptic pair anyway, and using "jellyfish-testing" for production doesn't sounds right.

Reply:
im on the fence on this one. no idea how the future will look